composer.json

```diff
@@ -60,11 +60,16 @@
             },
             "drupal/path_redirect_import": {
                 "3395257 - #3 - Change CSV file upload location to private.": "https://www.drupal.org/files/issues/2023-10-19/3395257-change-csv-file-3_0.patch"
+            },
+            "drupal_git/migrate_process_trim": {
+                "3288638 - Automated Drupal 11 compatibility fixes": "https://git.drupalcode.org/project/migrate_process_trim/-/merge_requests/1.patch"
```

Comment:
You should avoid using the patches from merge requests as they could change as contributors continue working on them. It's better to 
1. copy the contents of that patch and store it in the repo OR
2. convert that file to a .patch file on Drupal.org and upload it as a comment on the issue.